setup.cfg

```diff
@@ -25,11 +25,13 @@ max-doc-length = 88
 ignore = W503, # line break before binary operator (https://black.readthedocs.io/en/stable/compatible_configs.html#flake8)
     E203,  # whitespace before ‘:’ (https://black.readthedocs.io/en/stable/compatible_configs.html#flake8)
     E231,  # missing whitespace after ‘,’, ‘;’, or ‘:’ (black formats this differently as well)
-    E501,  # too long lines - FIX THIS IN SEPARATE PR
     D100,  # ignore missing docstrings in public module
     D104,  # ignore missing docstrings in public package
     D105,  # ignore missing docstrings in magic methods
 docstring-convention = google
+per-file-ignores = tests/shared/nlu/training_data/formats/test_rasa_yaml.py:E501,
```

Comment:
why do we need these? Can't we just ignore individual lines in these files?

Reply:
These files have a lot of lines that cross the length limit, so I preferred to ignore the files at all. I thought about adding a `# flake8: noqa`, but it would ignore the other flake8 appoints, so I add the per-file-ignores. If you prefer I can update this to ignore individual lines.